build.gradle

```diff
@@ -51,6 +51,7 @@ ext {
 }
 
 def androidToolsVersion = '25.1.2'
+def butterknifeVersion = '8.4.0'
```

Comment:
I can rename it to "latestReleaseVersion"?

On Sep 6, 2016 8:21 AM, "Jake Wharton" <notifications@github.com> wrote:

> In build.gradle
> <https://github.com/JakeWharton/butterknife/pull/732#discussion_r77656411>
> :
>
> > @@ -51,6 +51,7 @@ ext {
> >  }
> >
> >  def androidToolsVersion = '25.1.2'
> > +def butterknifeVersion = '8.4.0'
>
> This should not be in the root build file. It conflates the meaning of
> what's in gradle.properties
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/JakeWharton/butterknife/pull/732/files/faae21fc9ac71b4368bf59bd1c43b6636c675b30#r77656411>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABqMSMj8LZPmYUfbFqARwPq9I--ixPCyks5qnYTcgaJpZM4J0V-F>
> .
>